docs/ManualDeploymentGuideScalarDLOnAzure-1.md

```diff
@@ -0,0 +1,227 @@
+
+3. Create the docker-registry secret for pulling the Scalar DL images from GitHub Packages.
+    
+   ```console
+    $ kubectl create secret docker-registry reg-docker-secrets --docker-server=ghcr.io --docker-username=<github-username> --docker-password=<github-personal-access-token>
+    ```
+   
+4. Run the Helm commands on the bastion to install Scalar DL on AKS.
+    
+   ```console
+    # Add Helm charts
+    $ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
+    
+    # List the Scalar charts.
+    $ helm search repo scalar-labs
+    
+    # Load Schema for Scalar DL install with a release name `load-schema`
+    $ helm upgrade --version <chart version> --install load-schema scalar-labs/schema-loading --namespace default -f schema-loading-custom-values.yaml
+   
+    # Install Scalar DL with a release name `my-release-scalardl`
+    $ helm upgrade --version <chart version> --install my-release-scalardl scalar-labs/scalardl --namespace default -f scalardl-custom-values.yaml
+   ```
+
+Note:
+
+* The same commands can be used to upgrade the pods.
+* Release name `my-release-scalardl` can be changed as per your convenience.
+* The `chart version` can be obtained from `helm search repo scalar-labs` output
+* `helm ls -a` command can be used to list currently installed releases.
+* You should confirm the load-schema deployment has been completed with `kubectl get pods -o wide` before installing Scalar DL.
+* Follow the [Maintain Scalar DL Pods](./MaintainPods-1.md) for maintaining Scalar DL pods with Helm.
+
+## Step 5. Monitor the cluster
+
+It is critical to actively monitor the overall health and performance of a cluster running in production.
+This section shows how to configure monitoring and logging for your AKS cluster.
+
+## Recommendations
+
+* You should configure alerting for the AKS cluster on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-metric-alerts#enable-alert-rules)
+
+### Steps
+
+* Enable monitoring of Azure Kubernetes Service on the basis of [the official guide](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-enable-existing-clusters).
+
+## Step 6. Checklist for confirming Scalar DL deployments
+
+After the Scalar DL deployment, you need to confirm that deployment has been completed successfully. This section will help you to confirm the deployment.
+
+### Confirm Scalar DL deployment
+
+* Make sure the schema is properly created in the underlying database service.
+
+* You can check if the pods and the services are properly deployed by running the `kubectl get pods,services -o wide` command on the bastion.
+    * You should confirm the status of all ledger and envoy pods are `Running`.
+    * You should confirm the `EXTERNAL-IP` of Scalar DL envoy service is created.
```

Comment:
It seems that the `EXTERNAL-IP` is not assigned (marked as `<pending>`) unless the `service.beta.kubernetes.io/azure-load-balancer-internal` setting in `schema-loading-custom-values.yaml` is changed to `false`. Is it correct? If so, can you describe this?

Reply:
Do you mean in `scalardl-custom-values.yaml` , 
>service.beta.kubernetes.io/azure-load-balancer-internal setting in schema-loading-custom-values.yaml is changed to false. Is it correct?

We dont need to change the setting to have the `EXTERNAL-IP`.